### PR TITLE
Init

### PR DESCRIPTION
**Pull request by FromLiQg**

### CODE REVIEW

File: SConscript [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import os
```
Comment (xujuntwt95329) [inline SConscript:7]: Should add license header

File: app-framework/app_ext_lib_export.c [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#include "lib_export.h"
```
Comment (xujuntwt95329) [inline app-framework/app_ext_lib_export.c:6]: Should add license header

Comment (FromLiQg) [reply app-framework/app_ext_lib_export.c:6]: Does this also need to add a license header?
@xujuntwt95329 

https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/app-framework/app_ext_lib_export.c

File: SConscript (hunk 1 above)
Comment (FromLiQg) [reply SConscript:7]: done

File: README.md [hunk 3]
```diff
@@ -1 +1,19 @@
-# wamr-app-framework
\ No newline at end of file
+# wamr-app-framework
```
Comment (wenyongh) [inline README.md]: Can we add some introduction in the README? Refer to introduction in WAMR README:


Useful components and tools for building real solutions with WAMR vmcore:
[App-framework](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/app-framework/README.md): A framework for supporting APIs for the Wasm applications
[App-manager](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/app-mgr/README.md): a framework for dynamical loading the Wasm module remotely
[WAMR-IDE](https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/test-tools/wamr-ide): An experimental VSCode extension for developing WebAssembly applications with C/C++

**Review by wenyongh — COMMENTED** [review]
Cannot show picture on:
https://github.com/FromLiQg/wamr-app-framework-1/tree/main/wamr-sdk#menu-configuration-for-building-sdk
Seems there is missing files:
https://github.com/FromLiQg/wamr-app-framework-1/blob/main/doc/pics/wamr_menu_config.png

**Comment by wenyongh** [comment]
Same in:
https://github.com/FromLiQg/wamr-app-framework-1/tree/main/app-mgr
https://github.com/FromLiQg/wamr-app-framework-1/tree/main/wamr-sdk

File: README.md (hunk 3 above)
Comment (FromLiQg) [reply README.md]: Done

**Review by wenyongh — APPROVED** [review]
LGTM

**Comment by xujuntwt95329** [comment]
LGTM

**Pull request merged**